src/ui/actions/timeline.ts

```diff
@@ -253,11 +253,7 @@ export function seek(
   hasFrames: boolean,
   pauseId?: PauseId
 ): UIThunkAction<boolean> {
-  return ({ getState, dispatch }) => {
-    if (!selectors.isRegionLoaded(getState(), time)) {
-      return false;
-    }
-
```

Comment:
Thinking about this more, I think the logic should read

```js
  return ({ getState, dispatch }) => {
    if (!selectors.isRegionLoaded(getState(), time)) {
      hasFrames = false;
      pauseId = null;
    }
```

That way we always seek to the correct point in time, but we dont try to perform actions that will fail

Reply:
I think we should try removing this altogether first. If we set `pauseId` to `null` here, a new Pause will be created for the given point, which may then be reused later when the region has been loaded again. In this case the frontend may falsely keep assuming that there are no frames for this point.